src/pages/index.js

```diff
@@ -13,11 +13,15 @@ class Homepage extends React.Component {
   }
 
   render() {
-    const siteTitle = "Bastion";
-    const fort_example = `#[fort::root]
-fn main() {
+    const siteTitle = "Bastion";    
```

Comment:
```suggestion
    const siteTitle = "Bastion";
```

Remove trailing whitespaces.